sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Renamed `GetUri` to `GetUrl`.
 - Added `DataLakeLeaseClient`, all lease related APIs are moved to `DataLakeLeaseClient`.
 - Changed lease duration to be `std::chrono::seconds`.
+- Renamed `DataLakePathClient::GetAccessControls` to `DataLakePathClient::GetAccessControl`.
```

Comment:
When is this called `AccessControl` and when is it called `AccessControlList`?

Reply:
I don't know, I just align with .NET. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs#L1870

Reply:
whatever

Reply:
Changed to `GetAccessControlList` as agreed.